extensions/wikia/FounderEmails/maintenance.php

```diff
@@ -19,9 +19,11 @@
 }
 
 if ( isset( $options['help'] ) ) {
-	die( "Usage: php maintenance.php [--event=(daysPassed/viewsDigest/completeDigest)] [--wikiId=12345] [--help] [--quiet]
+	die( "Usage: php maintenance.php [--event=(daysPassed/viewsDigest/completeDigest)] [--wikiId=12345] [--test-days=3] [--help] [--quiet]
 		--event			specific email event i.e. daysPassed, viewsDigest, completeDigest
 		--wikiId		specific wiki id
+		--test-days     test daysPassed email with a specific day value.  Creates fake record that 
+		                will always trigger.  Requires --wikiId.
```

Comment:
Using `--test-days` allows us to test the daysPassed email.  A wiki ID is always required because we need to create a fake row for a specific wiki.